src/app/components/home-smorgasbord/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+class HomeSmorgasbord extends Component {
```

Comment:
Although you say this component will be more complex, as per today it can be a stateless component.